base/stats.go

```diff
@@ -61,6 +61,28 @@ const (
 
 const StatsGroupKeySyncGateway = "syncgateway"
 
+const (
+	PrometheusValueTypeGauge   = "gauge"
+	PrometheusValueTypeCounter = "counter"
+
+	StatUnitNoUnits       = ""
+	StatUnitPercent       = "percent"
+	StatUnitBytes         = "bytes"
+	StatUnitNanoseconds   = "nanoseconds"
+	StatUnitSeconds       = "seconds"
+	StatUnitUnixTimestamp = "unix timestamp"
+
+	StatFormatInt      = "int"
+	StatFormatFloat    = "float"
+	StatFormatDuration = "duration"
+	StatFormatBool     = "bool"
+
+	StatVersionAdded3dot1dot0 = "3.1.0"
+
+	StatStabilityNoStability = ""
+	StatStabilityCommitted   = "committed"
```

Comment:
If this stability value does turn out to be an enum, specify the other options here, even if we don't make use of them.

Reply:
As above